client/dashboard/domains/name-servers/styles.scss

```diff
@@ -0,0 +1,30 @@
+@import '@wordpress/base-styles/colors';
+
+.domains-management__name-servers {
+
+	&.components-card__body {
+		padding-top: 1.5rem;
```

Comment:
Are these padding CSS necessary, or can we just rely on Core cards?

Reply:
Sure, I removed the custom padding CSS.